test/test_xacro.py

```diff
@@ -63,17 +84,18 @@ def all_attributes_match(a, b):
 
     for a, b in zip(a_atts, b_atts):
         if a[0] != b[0]:
-            print("Different attribute names: %s and %s" % (a[0], b[0]))
+            print('Different attribute names: %s and %s' % (a[0], b[0]))
             return False
         if not text_values_match(a[1], b[1]):
-            print("Different attribute values: %s and %s" % (a[1], b[1]))
+            print('Different attribute values: %s and %s' % (a[1], b[1]))
             return False
     return True
 
 
 def text_matches(a, b):
-    if text_values_match(a, b): return True
-    print("Different text values: '%s' and '%s'" % (a, b))
+    if text_values_match(a, b):
+        return True
+    print('Different text values: \'%s\' and \'%s\'' % (a, b))
```

Comment:
For strings that have embedded single-quotes, we usually use double-quotes on the outside.